source/docs/06-neumes.xml

```diff
@@ -244,7 +244,7 @@
                   </figure>
                </p>
                <p>A further complication is that while the music encoding aims to narrow down and capture the meaning of the neumes in a logical and coherent system, occasionally the significance of some neumes is under debate (<abbr>e.g.</abbr>, <gi scheme="MEI">quilisma</gi>) and, despite its aim for accuracy, the encoding must remain open for future interpretations. From all these challenges has arisen the need for an early music encoding standardisation, that is, a set of rules that work for the description of any neume across all early notations regardless of the different methodologies applied to the study of individual notations and their idiosyncrasies.</p>
-               <p>Broadly speaking, Western early notations belong to two categories. On one side we have notations where two or more notes were represented by a single pen-stroke, while on the other side there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes. These distinctions have been described even within single notational styles as interrupted neumes `&lt;ineume&gt;` or uninterrupted `&lt;uneume&gt;` (Morent &amp; Schräder 2008; Morent 2011; MEI Guidelines v3, ch. 6: Neume Notation), and now as gapped or not gapped (Behrendt, Bain, &amp; Helsen 2017; MEI Guidelines v4, ch. 6: Neume Notation). To date, the MEI Neumes Module has been tested mainly on stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.</p>
```

Comment:
Since the reference to the literature is now removed, do you want to keep the 3 mentioned titles for uneumes and ineumes in the bibliography 6.5?

Reply:
Yes, please remove the 3 titles in 6.5. Thank you.